Makefile

```diff
@@ -26,7 +26,7 @@ YQ_INTERNAL := go run $(MOD_FLAGS) ./vendor/github.com/mikefarah/yq/v3/
 KUBEBUILDER_ASSETS := $(or $(or $(KUBEBUILDER_ASSETS),$(dir $(shell command -v kubebuilder))),/usr/local/kubebuilder/bin)
 export KUBEBUILDER_ASSETS
 GO := GO111MODULE=on GOFLAGS="$(MOD_FLAGS)" go
-GINKGO := $(GO) run github.com/onsi/ginkgo/ginkgo
+GINKGO := ginkgo
```

Comment:
go run github.com/onsi/ginkgo/ginkgo runs v1. Had to install the v2 cli locally and use that =(

Reply:
I think I found a way to execute it similar to before. I had to remove -mod=vendor though